<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="531" started-at="2019-02-08T08:30:07Z" finished-at="2019-02-08T08:30:07Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="531" started-at="2019-02-08T08:30:07Z" finished-at="2019-02-08T08:30:07Z">
      <class name="com.hrms.DataDriven">
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String)[pri:0, instance:com.hrms.DataDriven@7c0e2abd]" name="login" duration-ms="469" started-at="2019-02-08T14:00:07Z" data-provider="AdminServantMaidList" finished-at="2019-02-08T14:00:07Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
The data provider is trying to pass 3 parameters but the method com.hrms.DataDriven#login takes 2]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 3 parameters but the method com.hrms.DataDriven#login takes 2
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1337)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- com.hrms.DataDriven -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
